ci/test_cpp.sh

```diff
@@ -23,4 +23,9 @@ then
     popd
 fi
 
+if test "${TEST_LB_SCHEMA:-0}" -eq 1
+then
+    ${VT}/scripts/check_lb_data_files.sh
```

Comment:
To fix the Codacy / Shellcheck complaint:
```suggestion
    "${VT}/scripts/check_lb_data_files.sh"
```